ide-support/revsaveasiosstandalone.livecodescript

```diff
@@ -892,6 +892,18 @@ end revModifyMobileImageFile
 
 ################################################################################
 
+private command addToManifest pFile, pName, pAppBundle, @xManifest, @xRedirects
+   if there is a file pFile then
+      put pAppBundle & slash & tName & tab & pFile & return after xManifest
```

Comment:
I think this needs to be `pName` rather than `tName`.  It might be worth doing a sanity check to make sure that `pName` doesn't contain `tab` or `//`.